### DISCUSSION TITLE
Custom styling of default blocks

### DISCUSSION THREAD

**Discussion by BB-BenBridges** [quillforms/quillforms#165] [Q&A] [answered]
Is there a way I can edit the styling of the default blocks? I'd like to reduce the padding between blocks in a multiblock question.

**Comment by mdmag** [comment]
There is a way to add your own custom css easily

**Reply by BB-BenBridges** [reply]
Great! How can I do this?

**Reply by mdmag** [reply, marked as answer]
https://github.com/quillforms/quillforms/blob/master/react-docs/custom-css.md

**Reply by BB-BenBridges** [reply]
Thanks!
